src/tenant/admin.py

```diff
@@ -115,6 +127,79 @@ def get_queryset(self, request):
                     tenant.update_cached_fields()
         return qs
 
+    @admin.action(description="Send an email message to all owners for the selected tenant(s)")
+    def send_email_message(self, request, queryset):
+        """
+        Send an email message to all owners for selected tenant(s).
+
+        This action first displays an intermediate page which shows compose message form and
+        a list of all recipients.
+
+        Next, it send email message to all selected users and redirects back to the change list.
+        """
+        from siteconfig.models import SiteConfig
+
+        # get a list of selected tenant(s), excluding public schema
+        objects = self.model.objects.filter(
+            pk__in=[str(x) for x in request.POST.getlist(helpers.ACTION_CHECKBOX_NAME)]).exclude(
+                schema_name=get_public_schema_name())
+
+        # get a list of all owners (recipients) for selected tenant(s)
+        owners = []
+        for tenant in objects:
+            with tenant_context(tenant):
+                config = SiteConfig.get()
+                if config.deck_owner is not None:
+                    owners.append(config.deck_owner)
+        owners = set(owners)
```

Comment:
An explicit comment here would be helpful why you are using set() (to remove duplicates I assume?)

Reply:
It would be helpful and easier/faster for me to review comments if you replied with details instead of just marking them as resolved.  Otherwise I have to dig into the code which can delay my review for days.

In this case, thanks for the additional comment, except you say "...It first removes the duplicates and returns a dictionary which has....".  This is not correct. `set()` does not return a dictionary, it returns a `set` data structure.  You may have been confused because they both use {curly braces} in their syntax.

![image](https://github.com/bytedeck/bytedeck/assets/10604391/ae7999cb-6ea6-4e27-9918-997f5bb9d1ef)